base/docs/helpdb/Dates.jl

```diff
@@ -22,11 +22,12 @@ the new `DateTime` are assumed to be zero.
 """
 Dates.DateTime(dt::Date)
 
+
 """
     DateTime(dt::AbstractString, format::AbstractString; locale="english") -> DateTime
 
 Construct a `DateTime` type by parsing the `dt` date string following the pattern given in
```

Comment:
"Construct a DateTime type" is misleading (here and below), as an instance is constructed. While you're at it, could you fix this too? I think you'd better say "a `Date` or `DateTime` object", which is clearer.